### ISSUE TITLE
"failed to merge state file: 1 error occurred:"

### ISSUE THREAD

**Issue by mybayern1974** [Azure/aztfy#290]
Ran into this error when aztfy a resource group that only contains one resource that's an Azure Storage Account that includes several child containers and File Shares.

**Comment by magodo** [comment]
The issue is because the RG includes a `azurerm_storage_blob_inventory_policy` whose top level policy is disabled. Running `tf import` also failed:

```shell
💢  tf import azurerm_storage_blob_inventory_policy.test /subscriptions/xxxx/resourceGroups/yupstr/providers/Microsoft.Storage/storageAccounts/yupstr3/inventoryPolicies/default
azurerm_storage_blob_inventory_policy.test: Importing from ID "/subscriptions/xxxx/resourceGroups/yupstr/providers/Microsoft.Storage/storageAccounts/yupstr3/inventoryPolicies/default"...
azurerm_storage_blob_inventory_policy.test: Import prepared!
  Prepared azurerm_storage_blob_inventory_policy for import
azurerm_storage_blob_inventory_policy.test: Refreshing state... [id=/subscriptions/xxxx/resourceGroups/yupstr/providers/Microsoft.Storage/storageAccounts/yupstr3/inventoryPolicies/default]
╷
│ Error: Cannot import non-existent remote object
│
│ While attempting to import an existing object to "azurerm_storage_blob_inventory_policy.test", the provider detected that no object exists with the given id. Only pre-existing objects can be
│ imported; check that the id is correct and that it is associated with the provider's configured region or endpoint, or use "terraform apply" to create a new remote object for this resource.
```

That is because the provider is checking that property for existance:

https://github.com/hashicorp/terraform-provider-azurerm/blob/51c95924f60a426f844238b265a13fc9c0866baf/internal/services/storage/storage_blob_inventory_policy_resource.go#L228-L234

The remainig question is that why tf-exec import didn't fail...

**Comment by magodo** [comment]
Fixed by #298.
